src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -729,5 +733,25 @@ private static IEnumerable<string> GetClassesForType(Type type)
 				}
 			}
 		}
+
+		private InputCursor _protectedCursor = InputSystemCursor.Create(InputSystemCursorShape.Arrow);
```

Comment:
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.uielement.protectedcursor?view=windows-app-sdk-1.3#microsoft-ui-xaml-uielement-protectedcursor

Documentation says defaults to null.